HyprMXAdapter/src/main/java/com/chartboost/mediation/hyprmxadapter/HyprMXAdapter.kt

```diff
@@ -298,6 +298,33 @@ class HyprMXAdapter : PartnerAdapter {
         else -> ConsentStatus.CONSENT_STATUS_UNKNOWN
     }
 
+    /**
+     * Get a stored age restriction.
+     *
+     * @param context a context that will be passed to the SharedPreferences to get the age restriction.
+     *
+     * @return An already stored age restriction.
+     */
+    private fun getAgeRestricted(context: Context) =
+        context.getSharedPreferences(
+            HYPRMX_PREFS_KEY,
+            Context.MODE_PRIVATE
+        ).getBoolean(HYPRMX_AGE_RESTRICTION_KEY, false)
+
+    /**
+     * Store a HyprMX age restriction value and set it to HyprMX.
+     * This is passed on SDK initialization.
```

Comment:
What is passed on sdk init? Coppa? It's definitely set in set age restricted, not init. 
This method is called in init? That is not useful documentation for this method. This method could be called from anywhere that finds it useful.
Or do you mean coppa usually is set on sdk init like from mediation?

Reply:
Oh, yeah, let me rephrase this. Thanks for pointing it out. 👍🏼